app/lib/domains.server.ts

```diff
@@ -18,7 +18,7 @@ interface DomainRecord {
   ports?: Record['ports'];
 }
 
-type Domain = DnsRecord & DomainRecord;
+export type Domain = DnsRecord & DomainRecord;
 
 export const createUserDomain = async (data: Domain) => {
```

Comment:
Could we rename these to records? I strongly feel about the term `domain` is not correct here

Reply:
I was thinking an easy name for outsiders, but I think we should have a consistency as I look at the table and everything. I will update them all.